packages/web/src/utils/prerender.utils.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react'
+
+/* Web side prerender utils, to be used on the browser */
+
+export const isPrerendering = (): boolean => {
+  return !!global?.__REDWOOD_PRERENDER_MODE
+}
+
+export const useIsBrowser = () => {
+  return useMemo(() => {
+    return {
+      browser: !global?.__REDWOOD_PRERENDER_MODE,
+    }
+  }, [])
+}
+
+export const BrowserOnly = ({ children }: { children: React.ReactNode }) => {
+  const { browser } = useIsBrowser()
+
+  return browser && children
+}
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace NodeJS {
+    interface Global {
+      __REDWOOD_PRERENDER_MODE: boolean
```

Comment:
Nitpicky naming again, I'm sorry, maybe "MODE" isn't the best name here, maybe `__REDWOOD__PRERENDER_ENABLED` or `__REDWOOD__PRERENDERING`  since it's a boolean, I would expect mode to be a string that describes the type of prerendering.

Reply:
✅